src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.const.ts

```diff
@@ -0,0 +1,14 @@
+import i18n from 'i18n';
+
+const { t } = i18n;
+
+export const SLIDER_VALUE_HEAD_ROWS = [
+  {
+    id: 'placeholder',
+    label: t('value'),
+  },
+];
+
+export const SLIDER_LABEL_ROWS = [
```

Comment:
the same with a function instead of variable

Reply:
done